### PR TITLE
Updates DotNetNuke References

### PR DESCRIPTION
**Pull request by thabaum**
For Issue: https://github.com/dnnsoftware/Dnn.Platform/issues/3345
## Summary

Updates DotNetNuke CMS References in Host Template

### CODE REVIEW

File: DNN Platform/Website/Install/DotNetNuke.install.config.resources [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <script name="Schema">DotNetNuke.Schema</script>
     <script name="Data">DotNetNuke.Data</script>
   </scripts>
-  <version>08.00.04</version>
+  <version>09.04.04</version>
```
Comment (ohine) [inline DNN Platform/Website/Install/DotNetNuke.install.config.resources]: This variable is connected to the initial database installation process. If it's updated both `DotNetNuke.Schema.SqlDataProvider` and `DotNetNuke.Data.SqlDataProvider` will need to be updated to match the specified version. My recommendation would be to revert this single line back to v8.0.4.

Comment (thabaum) [reply DNN Platform/Website/Install/DotNetNuke.install.config.resources]: I changed it back, sorry I was not intending on changing that version number I had changed it back to original but may have undone the change while making changes using ctrl + z.  Thank you for the catch.

**Comment by thabaum** [comment]
changed it back however i see my PR's getting closed, Any Reason?

**Comment by bdukes** [comment]
@mitchelsellers did you intend to close this?

**Comment by bdukes** [comment]
Oh, @mitchelsellers deleted `release/9.4.x` which automatically closed this PR, since it was targeting that branch.  Can you recreate the PR against `develop`?  Thanks!

File: DNN Platform/Website/Install/DotNetNuke.install.config.resources (hunk 1 above)
Comment (thabaum) [reply DNN Platform/Website/Install/DotNetNuke.install.config.resources]: @ohine  when you going release that weather widget again??? I miss that module :)

**Comment by valadas** [comment]
Oh, this might be a little my fault :) It hough I had all the 9.4.x prs merged, looks like I forgot a couple before giving the go ahead to delete release/9.4.x. Sorry for that, unfortunately I cannot retarget it using the UI. @thabaum if you still have that branch you may be able to resubmit a new PR agains develop without conflicts (I think).

**Comment by valadas** [comment]
Yep, just tried it and it should work, you can go https://github.com/thabaum/Dnn.Platform/tree/patch-18 and the the button to submit a pull request and it looks conflict free. I could even do it, just want you to get your credit for this :)

**Comment by david-poindexter** [comment]
@valadas I think he’ll get credit for his commits.  Your switch should just add another commit by you.

**Comment by valadas** [comment]
Oh ok then. Thanks for the info, yeah, the commit is still from him.

**Comment by valadas** [comment]
Done resubmitted as #3393 loving our new branches :)

**Pull request closed without merging**